src/BCrypt/BCrypt+AlgorithmIdentifiers.cs

```diff
@@ -6,7 +6,11 @@ namespace PInvoke
     /// <content>
     /// Contains the <see cref="AlgorithmIdentifiers"/> nested class.
     /// </content>
-    public partial class BCrypt
```

Comment:
That's a good point. I've opened #37 to track. 

Originally I did this both to avoid them having to reference both assemblies to get at NCrypt, and because referencing BCrypt from NCrypt caused a subtle but very important build problem. But I have another idea how to solve that.